powerauth-fido2-tests/src/main/resources/templates/login.html

```diff
@@ -0,0 +1,105 @@
+                <div class="form-group">
+                    <input type="text" id="username" name="username" autoComplete="username webauthn" placeholder="Username" class="form-control" />
+                </div>
+
+                <!-- Info and login banners -->
+                <div id="errorDiv" class="alert alert-danger text-center" hidden="hidden">
+                    <span id="errorMessage"></span>
+                </div>
+                <div id="successDiv" class="alert alert-success text-center" hidden="hidden">
+                   Success
+                </div>
+
+                <!-- Submit buttons -->
+                <div class="form-group btn-group" style="width: 100%;">
+                    <button type="submit" id="loginBtn" class="btn btn-primary " style="width: 50%;">Log in</button>
+                    <button type="submit" id="registerBtn" class="btn btn-danger " style="width: 50%;">Register</button>
+                </div>
+
+                <!-- Advanced settings -->
+                <button type="button" id="settingsBtn" class="btn btn-primary btn-block btn-xs">Settings</button>
+                <div id="settingsBlock" class="info block" hidden="hidden">
+                    <label for="operationTemplate" class="label">Operation Template</label>
+                    <select id="operationTemplate" class="form-control">
+                        <option th:each="option : ${templates}" th:value="${option}" th:text="${option}" th:selected="${option} == login ? true : false"></option>
+                    </select>
+
+                    <label for="authenticatorAttachment" class="label">Authenticator Attachment</label>
+                    <select id="authenticatorAttachment" class="form-control">
+                        <option>platform</option>
+                        <option selected>all supported</option>
+                        <option>cross-platform</option>
+                    </select>
+
+                    <label for="residentKey" class="label">Discoverable Credential</label>
+                    <select id="residentKey" class="form-control">
+                        <option>required</option>
+                        <option selected>preferred</option>
+                        <option>discouraged</option>
+                    </select>
+
+                    <label for="userVerification" class="label">User Verification</label>
+                    <select id="userVerification" class="form-control">
+                        <option selected>required</option>
+                        <option>preferred</option>
+                        <option>discouraged</option>
+                    </select>
+
+                    <label for="attestation" class="label">Attestation</label>
+                    <select id="attestation" class="form-control">
+                        <option>enterprise</option>
+                        <option selected>none</option>
+                        <option>direct</option>
```

Comment:
Can we make the `direct` option as default? This is what we usually need for testing. Thanks.